sites/fast-website/src/app/components/section-header/section-header.template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from "@microsoft/fast-element";
+import { SectionHeader } from "./section-header";
+
+export const SectionHeaderTemplate = html<SectionHeader>`
+    <template>
+        <slot name="heading"></slot>
+        <p part="short">${x => x.short}</p>
+        <p part="body">${x => x.body}</p>
```

Comment:
How is this different than "short"? Should this be a slot?

Reply:
The "body" is the "description" on this particular component. I am ok with making it a slot. This element looks to be the same whether used as a hero section or a page section, that was the reason I didn't initially go with a slot.

Reply:
Not prescriptive here, just asking the questions :)

Reply:
Also, should be a slot for the same reason as above if possible.